README.md

```diff
@@ -54,7 +54,42 @@ Text(
 )
 ```
 
+### Buttons
 
+Two button components are available - `SourceButton` and `SourceIconButton`.
+
+The buttons are have preset style and size variants corresponding to Source presets. Both Core and Reader Revenue themes are also provided.
+
+Using `SourceButton`:
+```kotlin
```

Comment:
TIL you can find out all the programming languages Github supports syntax highlighting for in markdown here https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L3618

It even has different highlighting for .kts and .gradle.kts files. Neat.